third_party/arcs/proto/BUILD

```diff
@@ -36,5 +36,9 @@ proto_library(
 
 cc_proto_library(
     name = "manifest_cc_proto",
-    deps = [":manifest_proto"],
+    protos = [
```

Comment:
Do we need to enumerate all the protos? As discussed offline, I think what we are already doing the right thing. I believe the only issue is that we should not invoke `bazel build` on a `cc_proto_library` target directly (which we do in `main.yml`).

The suggested fix is to add a `cc_library` rule that depends on the `cc_proto_library` rule and use it:

```
cc_library(
  name = "manifest"
  deps = [":manifest_cc_proto"]
)
```
Can you check if that is the case?

Reply:
Checked. As we discussed offline, it turns out that this is a solution to a slightly different problem.